critiquebrainz/frontend/templates/bb_literary_work/entity.html

```diff
@@ -60,6 +60,52 @@ <h4 style="margin-bottom:0;">{{ _('Reviews') }}</h4>
         {% endif %}
       </ul>
     {% endif %}
+
+    {% if work_rels_info is defined and work_rels_info %}
```

Comment:
because we always pass this variable into the template context in the view, we don't need the `is defined` check. Let's just use `{% if work_rels_info %}` (so it will omit this block if the variable is `[]` but show it otherwise)